docker/README.md

```diff
@@ -16,34 +16,36 @@ Supported architectures are:
 - `armv7` (Ex. Raspberry PI)
 - `arm64` (Ex. OrangePI NanoPI)
 
-## Tags
+> [!NOTE] If you get the error `standard_init_linux.go:207: exec user process caused "exec format error"`, you most likely installed the wrong package intended for a [different architecture](https://github.com/zwave-js/zwavejs2mqtt/tree/master/docs/troubleshooting/improper-arch.md).
 
-Supported tags are:
+## Available Tags
+
+Available tags are:
 
 - `latest` for the latest official release.
 - `master` newest version, image gets built after every new commit to the master branch in the [zwavejs2mqtt](https://github.com/zwave-js/zwavejs2mqtt/commits/master) repository. (not recommended for the average user)
 - `sha-<commit-sha>` (example: `sha-92d502a`)
 - `<version>` (example: `2.1.0`)
 
-Note: `dev` tag have been deprecated.
+Note: The `dev` tag has been deprecated.
 
-## Install
+## Installation
 
 There are three different way to start the container and provide data persistence. For all of the methods **remember to**:
 
 1. Replace `/dev/ttyACM0` with your serial device
-2. Add `-e TZ=Europe/Stockholm` (or your timezone) to the `docker run` command to set the correct timezone in container
-3. If you are using zwave-js WS server and you chance the default WS port, replace `3000:3000` with the port chosen in settings
+2. Add `-e TZ=Europe/Stockholm` `-e TZ=America/New_York` (or your timezone) to the `docker run` command to set the correct timezone in the container. Time zone codes can be found at: <https://en.wikipedia.org/wiki/List_of_tz_database_time_zones>.
```

Comment:
Two timezones?

Reply:
Maybe just two examples

Reply:
Then should there be an "or" in between the strings?